gnat/vss_config.gpr

```diff
@@ -28,10 +28,16 @@ abstract project VSS_Config is
    Build_Mode : VSS_Build_Kind :=
      external ("VSS_BUILD_MODE", external ("BUILD_MODE", "prod"));
 
+   Superproject :=
+     external ("VSS_SUPERPROJECT", external ("SUPERPROJECT", ""));
```

Comment:
I don't think we really need a batch of `XXX_SUPERPROJECT` (per project setting) currently.
I propose to remove them and keep just one `SUPERPROJECT`